imgix/react-imgix
# fix: width query param overrides in srcSet

## Description

This PR fixes a bug introduced in #395

The refactoring done there failed to properly override the `w` query parameter in `srcSet` if it was being passed with the src url.

## Checklist
### Bug Fix

- [ ] Update or add any necessary API documentation

## Steps to Test

Code:

```js
<Imgix src="https://assets.imgix.net/examples/pione.jpg?w=333" />
```

v8.6.2:
```html
<img
  src="https://assets.imgix.net/examples/pione.jpg?w=333&amp;auto=format&amp;fit=crop&amp;ixlib=react-8.6.2"
  srcset="https://assets.imgix.net/examples/pione.jpg?w=333&amp;auto=format&amp;fit=crop&amp;ixlib=react-8.6.2&amp;w=100 100w, https://assets.imgix.net/examples/pione.jpg?w=333&amp;auto=format&amp;fit=crop&amp;ixlib=react-8.6.2&amp;w=116 116w, ..."
>
```

this branch:
```html
<img
  src="https://assets.imgix.net/examples/pione.jpg?w=333&amp;auto=format&amp;fit=crop&amp;ixlib=react-8.6.2"
  srcset="https://assets.imgix.net/examples/pione.jpg?auto=format&amp;fit=crop&amp;ixlib=react-8.6.2&amp;w=100 100w, https://assets.imgix.net/examples/pione.jpg?auto=format&amp;fit=crop&amp;ixlib=react-8.6.2&amp;w=116 116w, ..."
>
```
Steps:

1.  Go to https://codesandbox.io/s/elegant-joliot-i04qb
2.  Inspect the image
3.  Look at the srcSet
4. See the bug
